code/datums/armor/station/exploration.dm

```diff
@@ -0,0 +1,33 @@
+/datum/armor/exploration/soft
+	melee = 0.35
+	melee = MELEE_TIER_DEFAULT
```

Comment:
isnt this an overwrite to 0?

Reply:
yep
tiers are arbitrary. right now it's 0 but we can basically set it to any value, that's why everything is a #define

Reply:
Then why do you set melee to 0.35 **in the line above**

Reply:
oh FUCK that's a typo